ui/page/about_page.go

```diff
@@ -88,21 +88,62 @@ func (pg *AboutPage) Layout(gtx layout.Context) layout.Dimensions {
 func (pg *AboutPage) layoutRows(gtx layout.Context) layout.Dimensions {
 	w := []func(gtx C) D{
 		func(gtx C) D {
-			return components.EndToEndRow(gtx, pg.version.Layout, pg.versionValue.Layout)
+			return components.Container{
+				Padding: layout.Inset{
+					Top:    values.MarginPadding20,
+					Bottom: values.MarginPadding20,
+					Left:   values.MarginPadding16,
+					Right:  values.MarginPadding16,
+				},
+			}.Layout(gtx, func(gtx C) D {
```

Comment:
This can be added into a method and called for every instance where it is used/needed.